src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Dec2Hex.cs

```diff
@@ -0,0 +1,214 @@
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.Dec2HexTArg1, TexlStrings.Dec2HexTArg2 };
+        }
+
+        protected override bool CheckInvocation(CheckTypesContext context, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertAllValues(args);
+            Contracts.AssertValue(argTypes);
+            Contracts.Assert(args.Length == argTypes.Length);
+            Contracts.AssertValue(errors);
+            Contracts.Assert(MinArity <= args.Length && args.Length <= MaxArity);
+
+            var fValid = CheckInvocation(args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
+            if (argTypes.Length == 1)
+            {
+                var type = argTypes[0];
+                var arg = args[0];
+                if (!type.IsTable)
+                {
+                    errors.EnsureError(DocumentErrorSeverity.Severe, arg, TexlStrings.ErrTypeError);
+                    return false;
+                }
+            }
+            else if (argTypes.Length == 2)
+            {
+                var type0 = argTypes[0];
+                var type1 = argTypes[1];
+                var arg1 = args[0];
+                var arg2 = args[0];
+
+                if (!type0.IsTable && !type1.IsTable)
+                {
+                    errors.EnsureError(DocumentErrorSeverity.Severe, arg1, TexlStrings.ErrTypeError);
+                    errors.EnsureError(DocumentErrorSeverity.Severe, arg2, TexlStrings.ErrTypeError);
+                    return false;
+                }
+
+                var otherType = DType.Invalid;
+                TexlNode otherArg = null;
+
+                if (type0.IsTable)
+                {
+                    // Ensure we have a one-column table of numerics
+                    fValid &= CheckNumericColumnType(type0, args[0], errors, ref nodeToCoercedTypeMap);
+
+                    returnType = DType.CreateTable(new TypedName(DType.String, new DName(ColumnName_ValueStr)));
+
+                    // Check arg1 below.
+                    otherArg = args[1];
+                    otherType = type1;
+                }
+                else if (type1.IsTable)
+                {
+                    // Ensure we have a one-column table of numerics
+                    fValid &= CheckNumericColumnType(type1, args[1], errors, ref nodeToCoercedTypeMap);
+
+                    // Since the 1st arg is not a table, make a new table return type
+                    // *[Value:n] if the consistent return schema flag is enabled
+                    returnType = DType.CreateTable(new TypedName(DType.String, new DName(ColumnName_ValueStr)));
```

Comment:
>returnType = DType.CreateTable(new TypedName(DType.String, new DName(ColumnName_ValueStr))); [](http://example.com/codeflow?start=20&length=92)

This can be done outside the `if` since it's the same for both branches